Samples.xcodeproj/project.pbxproj

```diff
@@ -714,6 +719,14 @@
 			path = "Sketch on map";
 			sourceTree = "<group>";
 		};
+		D734FA072A183A5A00246D7E /* Set max extent */ = {
+			isa = PBXGroup;
+			children = (
+				D734FA092A183A5B00246D7E /* SetMaxExtentView.swift */,
+			);
+			path = "Set max extent";
+      sourceTree = "<group>";
```

Comment:
The indentation here is off due to that we manually resolved conflicts. To fix this, simply sort the samples by name again, and Xcode will correct this kind of errors.